Resources/Prototypes/Hydroponics/seeds.yml

```
@@ -1336,6 +1337,36 @@
       Max: 10
       PotencyDivisor: 10
 
+- type: seed
+  id: ambrosiaHorribilis
+  name: seeds-ambrosiahorribilis-name
+  noun: seeds-noun-seeds
+  displayName: seeds-ambrosiahorribilis-display-name
+  plantRsi: Objects/Specific/Hydroponics/ambrosia_horribilis.rsi
+  packetPrototype: AmbrosiaHorribilisSeeds
+  productPrototypes:
+    - FoodAmbrosiaHorribilis
+  lifespan: 5
```

```suggestion
  lifespan: 25
```
copy paste error?
Its lifespan is shorter than the maturation time

---

i was trying to see if i could make its lifespan really low to "balance" it but ingame lifespan seems to do nothing if set really low